cluster/ci/config/mirroring/origin_v3_9

```diff
@@ -1,17 +0,0 @@
-docker.io/openshift/origin:v3.9 quay.io/openshift/origin:v3.9 quay.io/openshift/origin-control-plane:v3.9
```

Comment:
Why are these deleted?

Reply:
All 3.x ones have to be preserved.

Reply:
They are only removed because I made copies in an earlier PR, the copies are in https://github.com/openshift/release/tree/master/core-services/quay-mirroring/openshift, including the 3.x ones. I'm keeping the copies in sync in this PR.

The only special one is the 3.9 one - that file exists but there was no job that used it. So I made the copy to not follow the mapping name convention, so it is not actually picked by the mirroring job, but we still keep that file.

Reply:
Ok